helm_deploy/values-cp-dev.yaml

```diff
@@ -10,6 +10,28 @@ generic-service:
     enabled: true
     host: prison-api-dev.prison.service.justice.gov.uk
     tlsSecretName: prison-api-cert
+    modsecurity_enabled: true
+    # Missing User Agent Header - 920320
+    # Request Missing an Accept Header - 920300
+    # Restrict file extension - 920440
+    # SQL Injection Attack: Common Injection Testing Detected - 942110
+    # Found User-Agent associated with scripting/generic HTTP client - 913101
+    modsecurity_snippet:
+      SecRuleEngine On
+      SecDefaultAction "phase:2,pass,log,tag:github_team=dps-tech"
+      SecRuleRemoveById 920320
+      SecRuleRemoveById 920300
+      SecRuleRemoveById 920440
+      SecRuleRemoveById 942110
+      SecRuleRemoveById 913101
+      SecRuleUpdateTargetById 932150 "!ARGS:/keywords/"
```

Comment:
copied exclusions from https://github.com/ministryofjustice/nomis-api-terraform-azure/blob/main/appgw-v2.tf, looked for anything that isn't auth related